pkg/strategy/streaming.go

```diff
@@ -94,6 +94,7 @@ func newStreamingStrategy(ctx context.Context, jaeger *v1.Jaeger) S {
 
 	// add autoscalers
 	manifest.horizontalPodAutoscalers = collector.Autoscalers()
+	manifest.horizontalPodAutoscalers = append(manifest.horizontalPodAutoscalers, ingester.Autoscalers()...)
```

Comment:
Can't we replace these two lines with `manifest.horizontalPodAutoscalers = append(collector.Autoscalers(), ingester.Autoscalers()...)`

Reply:
Sure we can. +1